packages/fileeditor-extension/src/commands.ts

```diff
@@ -98,6 +103,12 @@ export namespace CommandIDs {
   export const selectAll = 'fileeditor:select-all';
 }
 
+export interface IExtensionData extends ReadonlyJSONObject {
```

Comment:
"Extension" unfortunately is so overloaded in JupyterLab that perhaps we should make this name more explicit, like `IFileTypeData` or `IFileData` or `IFileExtensionData`?